package.json

```
@@ -1,8 +1,13 @@
 {
   "name": "svelte-i18n",
   "version": "3.7.4",
-  "main": "dist/runtime.cjs.js",
-  "module": "dist/runtime.esm.js",
+  "type": "module",
+  "main": "dist/runtime.js",
+  "exports": {
+    "types": "./dist/runtime.d.ts",
+    "import": "./dist/runtime.js",
+    "require": "./dist/runtime.cjs"
```

I'd personally make it just:
```suggestion
    "default": "./dist/runtime.js"
```

---

Yes and it would work also just adding the `"default": "./dist/runtime.js"` line because it is just a fallback. All this stuff is documented [here](https://nodejs.org/api/packages.html#conditional-exports).

I personally do not need of cjs so I do not care on if this package is a "Dual CommonJS/ES module package" or just esm. 

@kaisermann will split this PR and then he will decide on keeping or not the cjs.

---

I think we can be esm-only from v4 onwards, assuming `3.7.5` will be working as expected.